main.tf

```diff
@@ -44,6 +44,12 @@ locals {
     }
   )
   node_group_tags = merge(local.node_tags, local.autoscaler_enabled ? local.autoscaler_tags : null)
+
+  windows_taint = [{
+    key    = "OS"
+    value  = "Windows"
+    effect = "NO_SCHEDULE"
+  }]
```

Comment:
A taint like this, with a non-standard key, should at least be optional. I would argue it should be supplied by the user if desired, because they may already be using a different key for tainting by OS or may not want to taint at all and instead rely on node selectors with the ["well-known"](https://kubernetes.io/docs/reference/labels-annotations-taints/#kubernetes-io-os) `kubernetes.io/os` label.

Reply:
You'd use the nodeselector but the taint stops the mass of ingress/nginx/haproxy/grafana helm default installs from pointlessly trying to install on the nodes. 
I did leave it out originally and just had it in the docs. 🤷‍♀️